errors/errcode.go

```diff
@@ -60,6 +61,7 @@ var ErrCode2Str = map[ErrCode]string{
 	ErrDuplicateSubscription: "Duplicate subscription in one block",
 	ErrSubscriptionLimit:     "Subscription limit exceeded in one block",
 	ErrDoNotPropagate:        "Transaction should not be further propagated",
+	ErrAlreadySubscribed:     "Client already subscribed to topic",
 }
```

Comment:
We are consider to abandon the error code as it looks helpless to make the code better  but just lost lots of detail of the error,   Does this error code used by outside SDK else?

Reply:
Yes, I plan to use this error in SDK to ignore it because it's not critical. But still error, because no tx were created.